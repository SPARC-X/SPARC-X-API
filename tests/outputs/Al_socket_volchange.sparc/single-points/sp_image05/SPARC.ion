# Ion File Generated by SPARC ASE Calculator
# 
# ASE-SORT:
# 0 1 2 3
# END ASE-SORT

ATOM_TYPE: Al
N_TYPE_ATOM: 4
PSEUDO_POT: 13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
COORD:
0.08337991094040 -0.06252539702939 0.45934918180202
-0.04763850834288 3.84740866294769 4.12574099587526
3.65487538182834 -0.11180312494937 3.86214727653533
3.76435901708534 3.60129555981306 -0.03871604946277

