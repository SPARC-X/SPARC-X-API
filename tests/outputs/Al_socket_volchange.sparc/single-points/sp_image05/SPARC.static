***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Al:
      0.0108945058       0.9918303666       0.0600190417
      0.9937755030       0.5027064158       0.5390736078
      0.4775498172       0.9853916875       0.5046321784
      0.4918550628       0.4705490219       0.9949413207
Total free energy (Ha): -9.044014572035739E+00
Atomic forces (Ha/Bohr):
 -6.4383507156E-03  -1.7503934315E-03  -3.5140925546E-03
 -2.2271019282E-03  -5.5990101886E-03   4.4954429082E-04
  5.9977955350E-03   1.9666995689E-03   2.2135453683E-03
  2.6676571088E-03   5.3827040511E-03   8.5100289551E-04
Stress (GPa): 
 -1.3874286734E+01  -2.0542245214E-01  -6.5598722945E-01 
 -2.0542245214E-01   1.0199279100E+01  -9.1615449397E-02 
 -6.5598722945E-01  -9.1615449397E-02  -2.9918239595E+01
