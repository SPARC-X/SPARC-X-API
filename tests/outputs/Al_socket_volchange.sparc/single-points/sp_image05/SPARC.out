***************************************************************************
*                       SPARC (version Oct 31, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Thu Jan 18 08:03:30 2024                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
LATVEC_SCALE: 1 1 1 
LATVEC:
7.653390809639560 0.000000000000000 0.000000000000000 
0.000000000000000 7.653390809639560 0.000000000000000 
0.000000000000000 0.000000000000000 7.653390809639560 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.000950043469
EXCHANGE_CORRELATION: GGA_PBE
NSTATES: 12
CHEB_DEGREE: 17
CHEFSI_BOUND_FLAG: 0
CALC_STRESS: 1
MAXIT_SCF: 100
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-03
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-05
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-06
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 2.60E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0
MIXING_PARAMETER: 1
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
NUM_CHEFSI: 1
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_ENERGY_DENSITY: 0
OUTPUT_FILE: SPARC
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors (Bohr):
7.653390809639560 0.000000000000000 0.000000000000000 
0.000000000000000 7.653390809639560 0.000000000000000 
0.000000000000000 0.000000000000000 7.653390809639560 
Volume: 4.4829270488E+02 (Bohr^3)
Density: 2.4074929800E-01 (amu/Bohr^3), 2.6978060734E+00 (g/cc)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 2
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 1 2
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  2
Mesh spacing                       :  0.510226 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  SPARC.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  Al 3
Pseudopotential                    :  13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
Atomic mass                        :  26.9815385
Pseudocharge radii of atom type 1  :  7.14 7.14 7.14 (x, y, z dir)
Number of atoms of type 1          :  4
Estimated total memory usage       :  3.01 MB
Estimated memory per processor     :  1.50 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2617264380E+00        1.835E-01        0.020
2            -2.2635507498E+00        2.209E-01        0.007
3            -2.2613426427E+00        1.061E-01        0.006
4            -2.2611876463E+00        8.196E-02        0.006
5            -2.2610529346E+00        4.572E-02        0.007
6            -2.2610175579E+00        2.358E-02        0.006
7            -2.2610111579E+00        1.528E-02        0.006
8            -2.2610042948E+00        1.401E-03        0.006
9            -2.2610036330E+00        1.247E-03        0.006
10           -2.2610036430E+00        5.545E-04        0.006
Total number of SCF: 10    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2610036430E+00 (Ha/atom)
Total free energy                  : -9.0440145720E+00 (Ha)
Band structure energy              :  5.5282830452E-01 (Ha)
Exchange correlation energy        : -4.3629942646E+00 (Ha)
Self and correction energy         : -1.2498708777E+01 (Ha)
-Entropy*kb*T                      : -3.1904180229E-03 (Ha)
Fermi level                        :  1.2939237025E-01 (Ha)
RMS force                          :  6.5849196184E-03 (Ha/Bohr)
Maximum force                      :  7.5408940839E-03 (Ha/Bohr)
Time for force calculation         :  0.014 (sec)
Pressure                           :  1.1197749077E+01 (GPa)
Maximum stress                     :  2.9918239595E+01 (GPa)
Time for stress calculation        :  0.024 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  0.199 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*               List of contributors: See the documentation               *
*         Citation: See README.md or the documentation for details        *
*  Acknowledgements: U.S. DOE SC (DE-SC0019410), U.S. DOE NNSA (ASC)      *
*      {Preliminary developments: U.S. NSF (1333500,1663244,1553212)}     *
***************************************************************************
                                                                           
